### ISSUE TITLE
Add option to allow cross origin for builder

### ISSUE THREAD

**Issue by TPreece101** [streamsync-cloud/streamsync#65]
I've been trying this out locally and it works great, I'm having lots of trouble when I try to use a remote environment however.

For example if you try to run streamsync in Github Codespaces you cannot connect to the builder due to the request origin restriction.

Instructions to re-produce:
1. Open a repo with a streamsync app in Github Codespaces
2. Install requirements
3. Run app with `streamsync edit <path_to_folder>`
4. Try to open the builder by clicking "Open with Browser" on the proxy popup

You will get a `Connection rejected.` on the page and the following in the terminal:
```sh
ERROR:root:A session request with origin https://xxxxxx-3006.preview.app.github.dev was rejected. For security reasons, only local origins are allowed in edit mode.
```

I understand the need for this to be the default but it would be great to be able to turn this off if you're handling the auth separately and you're confident that it is secure.

For example it would be great to add an argument so that you could do something like:

```sh
streamsync edit --allow-cross-origin <my_path>
```
And then it could flash up a warning whilst it's running that looks something like:
```sh
WARNING: Allowing cross origin is not recommended - only use this argument if you're securing the end point with another service
```
 
Being able to edit these apps in Github Codespaces would be great way to test out the library with minimal local setup required.

Let me know what you think 😊

**Comment by ramedina86** [comment]
Hi @TPreece101, this makes total sense. I couldn't think of a scenario where I'd want to allow this, but Github Codespaces is a good one. I'll implement this logic for the next release.

**Comment by ramedina86** [comment]
`--enable-remote-edit` is available in v0.1.12, which was just released. I didn't go for "cross origin" as it makes me think of CORS. Thanks for sharing this.

**Comment by TPreece101** [comment]
No problem, thanks for implementing it so speedily 😊
